plugin/src/location.ts

```diff
@@ -1,8 +1,8 @@
-import GoogleAnalyticstModal from "@web/extensions/location/modals/googleAnalyticsModal";
-import TerrainModal from "@web/extensions/location/modals/terrainModal";
-import { PostMessageProps, MouseEventData } from "@web/extensions/location/types";
+import { PostMessageProps, MouseEventData } from "@web/extensions/location/core/types";
 
 import html from "../dist/web/location/core/index.html?raw";
+import GoogleAnalyticsHtml from "../dist/web/location/modals/googleAnalytics/index.html?raw";
+import TerrainHtml from "../dist/web/location/modals/terrain/index.html?raw";
```

Comment:
Keeping to the same conventions setup already, since these two are html, not actual React components, they should be camelCase.

Reply:
Done